packages/playground/blueprints/public/blueprint-schema.json

```diff
@@ -196,6 +196,9 @@
 						},
 						"prepareForRunningInsideWebBrowser": {
 							"type": "boolean"
+						},
+						"addFetchNetworkTransport": {
```

Comment:
Does this need a rebase?

Reply:
No it doesn't, that field is indeed missing from the schema. We should add a CI check for that.

Reply:
@adamziel This was an artifact of running `npm run build` before changing branches. Its been fixed.